dw-icon-button.js

```diff
@@ -70,6 +70,11 @@ export class DwIconButton extends buttonFocus(LitElement) {
           overflow: hidden;
           border-radius: 50%;
         }
+        
+        /* ripple color shows is to light so change a opcity to color. */
+        dw-ripple {
+          opacity: 1;
```

Comment:
@shakil-mansuri-dream I don't understand this change. Let's discuss it in person.